packages/fxa-settings/src/components/BentoMenu/index.test.tsx

```diff
@@ -3,41 +3,39 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import React from 'react';
-import { render, screen, fireEvent, act } from '@testing-library/react';
+import { render, screen, fireEvent } from '@testing-library/react';
 import BentoMenu from '.';
 
 describe('BentoMenu', () => {
+  const dropDownId = 'drop-down-bento-menu';
+
   it('renders and toggles as expected with default values', () => {
     render(<BentoMenu />);
 
     const toggleButton = screen.getByTestId('drop-down-bento-menu-toggle');
-    const dropDown = screen.queryByTestId('drop-down-bento-menu');
 
     expect(toggleButton).toHaveAttribute('title', 'Firefox Bento Menu');
-    expect(toggleButton).toHaveAttribute(
-      'aria-controls',
-      'drop-down-bento-menu'
-    );
+    expect(toggleButton).toHaveAttribute('aria-label', 'Firefox Bento Menu');
+    expect(toggleButton).toHaveAttribute('aria-haspopup', 'menu');
     expect(toggleButton).toHaveAttribute('aria-expanded', 'false');
-    expect(dropDown).not.toBeInTheDocument();
+    expect(screen.queryByTestId(dropDownId)).not.toBeInTheDocument();
```

Comment:
Probably worth rebasing against `main` so this doesn't show up in the diff, I think you fixed this in #13240